.travis.yml

```diff
@@ -85,6 +85,9 @@ jobs:
         - bundle install
         - bundle exec rake assets:precompile RAILS_ENV=production
         - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-        - docker build -t gatesso/gate:$TRAVIS_TAG -t gatesso/gate:latest .
+        - docker build -t gatesso/gate:latest .
+        - if [ -n "$TRAVIS_TAG" ]; then
+            docker tag gatesso/gate:latest gatesso/gate:$TRAVIS_TAG;
```

Comment:
If I understand the logic correctly, isn't this means only tagged commit will be tagges as 'latest' at the docker hub?
perhaps `docker tag gatesso/gate:latest` should be outside of this conditional block?